front/src/applications/editor/tools/rangeEdition/components/RangeEditionLeftPanel.tsx

```diff
@@ -43,14 +42,13 @@ const RangeEditionLeftPanel = () => {
       initialEntity,
       trackSectionsCache,
       selectedSwitches,
+      isSpeedRestriction,
```

Comment:
You should not add this in the context, but keep it a state value like the isPsl. You can know whether the speedSection is a Ralen 30/60 if the speedSection has a prop on_route not undefined

Reply:
done